stix_shifter_modules/azure_sentinel/stix_translation/json/from_stix_map.json

```diff
@@ -56,15 +64,15 @@
   },
   "user-account": {
     "fields": {
-      "user_id": ["userStates.accountName", "processes.accountName"],
+      "user_id": ["userStates.accountName", "processes.accountName", "userStates.aadUserId"],
       "account_login": ["userStates.logonId"],
       "account_type": ["userStates.userAccountType"],
       "account_last_login": ["userStates.logonDateTime"]
     }
   },
   "software": {
     "fields": {
-      "name": ["vendorInformation.provider", "networkConnections.applicationName"],
+      "name": ["vendorInformation.provider", "networkConnections.applicationName", "hostStates.os"],
```

Comment:
remove `"networkConnections.applicationName", "hostStates.os"` from "name" and move them to `x-msazure-sentinel-alert`

Reply:
fixed